database/README.md

```diff
@@ -424,6 +424,39 @@ owners      | bytea[] | not null | surplus capturing jit order owner included in
 Indexes:
 - PRIMARY KEY: btree(`auction_id`)
 
+### jit\_orders
+
+Contains all JIT orders that solvers settled onchain.
```

Comment:
nit: Can you add a word on what a jit order is (or a link to our docs where this is explained)

Reply:
Added